src/docs/development/data-and-backend/networking.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Networking
+description: Internet network calls in Flutter.
+---
+
+## Cross-platform http networking
+
+The [`http`][] package provides the simplest way to issue http requests. This
+package is supported on Android, iOS, and the web.
+
+## Platform notes
+
+### Android
+
+Android apps must [declare their use of the internet][declare] in the Android
+manifest (`AndroidManifest.xml `):
+
+```
+<manifest xlmns:android...>
+ ...
+ <uses-permission android:name="android.permission.INTERNET" />
+ <application ...
+</manifest>
+```
+
+## Samples
+
+For a practical samples of various networking tasks (incl. fetching data,
+WebSockets, and parsing data in the background) see the 
+[networking cookbook(https://flutter.dev/docs/cookbook#networking).
```

Comment:
Change URL to a relative link: "/docs/cookbook#networking". Also, we generally put the urls at the end of the file, in alphabetical order.